source/usage-examples/distinct.txt

```diff
@@ -3,3 +3,45 @@ Retrieve Distinct Values of a Field
 ===================================
 
 .. default-domain:: mongodb
+
+You can retrieve a list of distinct values for a field across a
+collection by using the ``Distinct()`` method. 
+
+The following example specifies a query filter and a field name to the
+``Distinct()`` method, which matches documents in the ``movies``
+collection where the ``directors`` field contains "Natalie Portman" and
+returns distinct values of the ``title`` field from the matched documents. 
+
+.. include:: /includes/usage-examples/run-example-tip.rst
+
+.. literalinclude:: /includes/usage-examples/code-snippets/distinct.go
+   :start-after: begin distinct
+   :end-before: end distinct
+   :emphasize-lines: 4
+   :language: go
+   :dedent:
+
+Click `here <{+example+}/distinct.go>`__ to see a fully runnable example.
```

Comment:
On the staging link this is currently a 404. Will this be updated later?

Reply:
yes! once merged into the repo, the link will work